esbuild.js

```diff
@@ -77,8 +77,6 @@ const options = {
         },
         cssModules(
             {
-                includeFilter: /\.css$/,
-                excludeFilter: /@ckeditor\/|@fortawesome\/fontawesome-svg-core\/|styleHostOnly\.css|normalize\.css/,
```

Comment:
thats why we dit it, we dont need that ugly logic anymore as we rely on conventions